src/pages/activity.js

```diff
@@ -85,28 +84,26 @@ export default function Activity() {
   }, []);
```

Comment:
@pablojp The logic of the above two useEffects that is almost the same shouldn't be here, it should in the API logic probably using a `promiseAll` to retrieve both data and the parse it. And here only set the state referring to the UI.